charts/stable/tdarr/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 2.00.10
 description: Tdarr is a self hosted web-app for automating media library transcode/remux management and making sure your files are exactly how you need them to be in terms of codecs/streams/containers etc.
 name: tdarr
-version: 4.1.3
+version: 4.1.4
```

```suggestion
version: 4.2.0
```
This is not really a bugfix, but more of an added "feature", so a minor version bump is warranted.